libpod/define/pod_inspect.go

```diff
@@ -79,7 +79,7 @@ type InspectPodInfraConfig struct {
 	NoManageResolvConf bool
 	// DNSServer is a set of DNS Servers that will be used by the infra
 	// container's resolv.conf and shared with the remainder of the pod.
-	DNSServer []string
+	DNSServer []net.IP
```

Comment:
Safer to keep this a string, Golang can do wierd things marshalling net.IP to JSON under some circumstances.